app/services/course_search_service.rb

```diff
@@ -52,6 +52,12 @@ def sites_with_distance_from_origin
     # only want new and running sites
     new_and_running_sites = sites_with_status.where(site_status[:status].in(%w[new_status running]))
 
+    # only sites that have a locatable address
+    # there are some sites with no address1 or postcode that cannot be
+    # accurately geocoded. We don't want to return these as the closest site.
+    # This should be removed once the data is fixed
+    new_and_running_sites = new_and_running_sites.where(sites[:address1].not_eq("").or(sites[:postcode].not_eq("")))
```

Comment:
Guessing we in favuor of `or` vs `and`

Reply:
Yeah. We had a look at the data and it looks like if they have one of these then they should be pretty locatable.